core/cmd/eth_keys_commands.go

```diff
@@ -124,15 +124,27 @@ type EthKeyPresenter struct {
 }
 
 func (p *EthKeyPresenter) ToRow() []string {
+	eth := "Unknown"
```

Comment:
I wonder if sometime in the future we would like to make this `Unknown` or `Balance` check an actual part of the `String()` call on the assets.